### PR TITLE
Update the internal ACTNUM in the FieldPropsManager

### PR DESCRIPTION
**Pull request by joakim-hove**
When cells have been deactivated we need to call out to the property contanier with the new active cells map.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this opm-grid=428 please

**Comment by blattms** [comment]
ok, just for understanding this:
At this stage the fieldprops arrays are already compressed (to the extend that the Grid representation after parsing the deck is)? This second pass is just to catch cells that got inactive because of operations
in the simulator during setup.

**Comment by joakim-hove** [comment]
> At this stage the fieldprops arrays are already compressed (to the extend that the Grid representation after parsing the deck is)? This second pass is just to catch cells that got inactive because of operations
in the simulator during setup.

That is 100% correct!

**Comment by joakim-hove** [comment]
OK - this has received som MPI love and attention. Merge?

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
I realize the MPI calls are probably not in the way we should do it; my main MPI experience was from the time I was a bachelor - now I am divorced and the kids have gone off to university.

Nevertheless this little piece of code has worked on my machine for some time, I "need" to get it in to reduce the number of in-flight PRs for the 3D refactoring; it is 100% protected by the `#ifdef ENABLE_3DPROPS_TESTING` - so all in all I will put on my asbestos and merge it. When jenkins agrees.

File: ebos/eclcpgridvanguard.hh [hunk 1]
```diff
@@ -280,6 +283,27 @@ protected:
...
+#ifdef ENABLE_3DPROPS_TESTING
+        std::vector<int> actnum;
+        unsigned long actnum_size;
+        if (mpiRank == 0) {
+            actnum = Opm::UgGridHelpers::createACTNUM(*grid_);
+            actnum_size = actnum.size();
+        }
+
+#ifdef HAVE_MPI
+        MPI_Bcast(&actnum_size, 1, MPI_UNSIGNED_LONG, 0, MPI_COMM_WORLD);
```
Comment (blattms) [inline ebos/eclcpgridvanguard.hh]: please use `grid().comm().bcast(&actnum_size, 1, 0)``then you can skip the ifdef

**Comment by blattms** [comment]
I am wondering why this step is actually needed on all processors, But of course I have not the faintest idea how the size of the properties is in a parallel run. Should it the number of active cell in the global grid or the active cells in the local partition of the distributed grid. At least in the latter case there
is no need to broadcast here if you do the compression only on rank 0.

**Pull request merged**